src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/toolbar.scss

```diff
@@ -5,23 +5,20 @@
     justify-content: space-between;
     height: $toolbarHeight;
     margin: 0;
-    background: $toolbarBackgroundColor;
-    box-shadow: 2px 6px 12px 0 rgba($shadowColor, .5);
 }
 
-.items {
-    display: inline-block;
-    list-style: none;
-    margin: 0;
-    padding: 0;
-    border-left: $itemsContainerBorder;
-    border-right: $itemsContainerBorder;
-    vertical-align: top;
-    background-color: $itemsBackgroundColor;
+.light {
+    background: $toolbarBackgroundLightColor;
+}
+
+.dark {
+    background: $toolbarBackgroundDarkColor;
+}
 
-    & > li {
-        display: inline-block;
-    }
+.toolbar {
```

Comment:
There is already a toolbar rule at the very top of this file, I would leave these definitions there :wink: